cmd/datamon/cmd/context.go

```diff
@@ -33,7 +33,7 @@ func init() {
 	rootCmd.AddCommand(ContextCmd)
 
 	contextTemplate = func() *template.Template {
-		const listLineTemplateString = `Model Version: {{.Version}}, Name: {{.Name}}, WAL: {{.WAL}}, ReadLog: {{.ReadLog}}, Blob: {{.Blob}}, Metadata: {{.Metadata}}, Version Metadata: {{.VMetadata}}`
+		const listLineTemplateString = `Model Version: {{.Version}}, Name: {{.Name}}, WAL: {{.WAL}}, ReadLog: {{.ReadLog}}, Blob: {{.Blob}}, Metadata: {{.Metadata}}, Version Metadata: {{.VMetadata}} {{printf "\n"}}`
```

Comment:
I think all this carriage return handling is not good: let's keep assuming we are using a standard logger which does that.
In a near future, I'd like template format to be a user-input (like say in docker CLI), so this will definitely surprise users.

Reply:
Hmm.. I will need to look into the templating by docker CLI, I guess there is a need for a graphQL equivalent for a CLI :-D

Reply:
docker commands have options like `--format '{{ .Image }}` and it's great to build scripts.

Reply:
> a graphQL equivalent for a CLI

what we're necessarily moving toward given that CLI is the user interface and we're implementing query capabilities.

i don't see this as a complexity explosion as of yet, and the motif of passing DSL as a string is the sort of escape hatch i'd suggest as well should the CLI wind up overly noisy/busy as a result.